public/static/docs/command-reference/import.md

```diff
@@ -160,14 +159,14 @@ deps:
 ```
 
 If the
-[Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+[Git-reference](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
```

Comment:
should we just use revisions here as well?

Reply:
or just explain it in a plain language - as people do more commits and/or push more data ... or as people release new models (and create new commits on top of the fixed one) ...

Reply:
Well, the hyphen was a typo (removed now) but this sentence (repeated in a couple docs) that goes "If the reference moves (e.g. branch)..." would not be correct with "revision". I don't think revisions move, they're basically commits. The reference however can go from pointing from one commit to another and that's what we're trying to say here.

Reply:
> or just explain it in a plain language

OK let me try this...

Reply:
See b30df290, but it's not a huge change.

Reply:
Not sure how to convey this in plain language, its a pretty technical note about the relationship between `rev` and the provided reference (branch, tag, etc.) But once I think once this terminology is updated in the core repo we'll get to update this once again in simpler terms.

Reply:
it's just too complicated. still. Even if we need to go and be precise for some reason it should be after we explain the essense in simple terms.

Reply:
OK but this paragraph was already complicated before this PR. The scope here is supposedly about `metrics diff` (and related terminology) so can we leave this for regular updates? I'll save it, won't forget.

Reply:
OK, you're right. I simplified it a lot and removed the jargon and links. The `--rev` option above will have more details anyway (BTW that is being updated in a different PR). See dc9fc23b.

Reply:
Oops wait. That commit I did it in the regular updates PR. Please review there and ignore here! See this https://github.com/iterative/dvc.org/pull/931#pullrequestreview-351849911. Resolving here.